src/mock-data/quote.ts

```diff
@@ -11,6 +11,14 @@ const quoteOutNigeriaCUSD: QuoteRequestBody = {
   country: 'NG',
 }
 
+const quoteInNigeriaCUSD: QuoteRequestBody = {
```

Comment:
Not a big deal but it looks like the only thing that's different here is the amount. Makes me wonder if we need a different quoteIn mock vs quoteOut mock

Reply:
It's probably not super necessary at this point, I'm good with either keeping it or removing it.